src/workerd/api/worker-rpc.c++

```diff
@@ -1870,7 +1871,10 @@ class JsRpcSessionCustomEventImpl::ServerTopLevelMembrane final: public capnp::M
     auto f = kj::mv(JSG_REQUIRE_NONNULL(
         doneFulfiller, Error, "Only one RPC method call is allowed on this object."));
     doneFulfiller = kj::none;
-    return capnp::membrane(kj::mv(target), kj::refcounted<CompletionMembrane>(kj::mv(f)));
+    auto completion = capnp::membrane(
+        kj::mv(target), kj::refcounted<CompletionMembrane>(kj::mv(f)));
+    return capnp::membrane(
```

Comment:
Should the RevokerMembrane wrap the target before or after `CompletionMembrane`? I'd think it'd be after, but `JsRpcSessionCustomEventImpl::sendRpc` does it before.

Reply:
Hmm, does this even belong inside `ServerTopLevelMembrane`, or should it be stacked on top?

I was expecting to see the `RevokerMembrane` added in `JsRpcSessionCustomEventImpl::run()`, stacked with `ServerTopLevelMembrane`. Though I guess I'm not sure if it makes a difference.